.circleci/config.yml

```
@@ -59,7 +59,7 @@ jobs:
       - install_common
       - setup_remote_docker
       - checkout
-      - run: ./gradlew build -x integrationTest --stacktrace
+      - run: ./gradlew javadoc && ./gradlew build -x integrationTest --stacktrace
```

run `javadoc` first to fail fast if there is doc formatting issue